spec/lib/pickup_bot/commands/set_time_spec.rb

```diff
@@ -83,13 +83,14 @@
 
   def next_tuesday
     today = Date.today
+    return output_date(today) if today.tuesday?
```

Comment:
haha did you have a CI local time sensitive failing test here?

Reply:
🤐

Reply:
BTW if today is tuesday, isn't **next tuesday** NEXT week and __this tuesday__ would be Date.today? Or am I misinterpreting this